views/components/Drawer.jsx

```diff
@@ -208,18 +210,29 @@ class DrawerComponent extends Component {
             href={`${basePath}/u/configure`}
           >
             <ListItemIcon>
-              <ColorLens style={{ color: '#97C0CE' }} />
+              <ColorLens style={{ color: dark_sky_blue }} />
             </ListItemIcon>
             <ListItemText primary="Configure" />
 
+          </ListItem>
+          <ListItem
+            button={true}
+            component='a'
+            href={`${basePath}/charts`}
```

Comment:
can we extract this route to a constant (`routes.charts`)?

Reply:
[89c25f1](https://github.com/AMP-SCZ/dpdash/pull/78/commits/89c25f103294e3e583dd2ca8d95164bd04b81388)